### ISSUE TITLE
Inconsistent signal remain in scene 

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#34201]
**Godot version:**
3.2 beta3
**OS/device including version:**
win7-64
**Issue description:**
I cant delete successful a old signal connection 
![image](https://user-images.githubusercontent.com/13235119/70391919-125af980-19d2-11ea-8fa2-75559d89065c.png)

here the tscn content:
[connection signal="input_event" from="." to="." method="_on_GunTurret_input_event"]
[connection signal="body_entered" from="DetectRadius" to="." method="_on_DetectRadius_body_entered"]

As you can see I find the old signal in editor but it do not exist in tscn
with error in debug:
E 0:00:03.102   emit_signal: Error calling method from signal 'input_event': 'KinematicBody2D(GunTurret.gd)::_on_Tank_input_event': Method not found..
  <C++ Source>  core/object.cpp:1228 @ emit_signal()

**Steps to reproduce:**
Clear/attach script  and move/rename scenes/script in project 
**Minimal reproduction project:**
NA

**Comment by capnm** [comment]
Can not reproduce.
**RMB -> disconnect** works even after you remove or rename the script from the node.

**Comment by git2013vb** [comment]
From my screen shot I understand if it came not from tscn content it came from somewhere else.
From where? In anther scene? That's my meaning to be inconsistent. 
It is possible the happen there are inherited scene. I forgot to mention it I use inherited scenes

**Comment by capnm** [comment]
Please attach a small project (zip) to reproduce.
I don't understand what exactly you think is broken.

**Comment by git2013vb** [comment]
> Please attach a small project (zip) to reproduce.
> I don't understand what exactly you think is broken.
I changed the project so I cant send the exactly uses case. Even because I notice this issue after a while.
So:
In the picture you see 2 names one is "Tank" and the other is "GunTurret" - as signal part of name;
The top one "_on_Tank_input_event()" is orphan. But I'm not sure if it come from an inherited scene.
It is possible when I tried to disconnect it the disconnect action didn't work in that inherited scene so it  show again and again?
